package.json

```diff
@@ -85,8 +85,8 @@
   "scripts": {
     "dev": "vite ./dev --port 3000",
     "build": "pnpm tsc && vite build ./dev --outDir ../public --emptyOutDir",
-    "test": "vitest --dom",
-    "test:coverage": "vitest run --dom --coverage",
+    "test": "NODE_ENV=production vitest --dom",
```

Comment:
Should these be production or test?

Reply:
@imolorhe test would be ideal, but then we would need to change `debug` util - i just want to hide the debugging output in tests, but not in vite

Reply:
fixed!